docs/concepts/withdrawals.md

```diff
@@ -9,8 +9,8 @@ sidebar_position: 8
 Validators staking ether on Mainnet after [The Merge](merge.md), accrue two
 forms of rewards:
 
-- Execution layer rewards paid directly to a withdrawal address (Ethereum address).
-- Consensus layer rewards for performing actions each epoch.
+- execution layer rewards paid directly to a withdrawal address (Ethereum address)
```

Comment:
Are you sure this line and the next should not start with the first word in initial UC?

Reply:
https://www.microsoft.com/en-us/microsoft-365-life-hacks/writing/punctuating-bullet-point-lists#:~:text=Be%20consistent,are%20following%20the%20same%20format.

Reply:
Sorry @mjsmike62 you were 100% correct!